_search-plugins/hybrid-search.md

```diff
@@ -1213,3 +1213,193 @@ The response contains scoring information:
 Field | Description
 :--- | :---
 `explanation` | The `explanation` object has three properties: `value`, `description`, and `details`. The `value` property shows the result of the calculation, `description` explains what type of calculation was performed, and `details` shows any subcalculations performed. For score normalization, the information in the `description` property includes the technique used for normalization or combination and the corresponding score. 
+
+## Paginate hybrid query results
+**Introduced 2.19**
+{: .label .label-purple }
+
+You can apply pagination to hybrid query results by using the `pagination_depth` parameter in the hybrid query clause, along with the standard `from` and `size` parameters. The `pagination_depth` parameter defines the maximum number of search results that can be retrieved from each shard per subquery. For example, setting `pagination_depth: 50` allows up to 50 results per subquery to be maintained in memory from each shard.
+
+To navigate through the results, use:
+- `from`: specifies the document number from which you want to start showing the results, default is `0`
+- `size`: specifies the number of results to return on each page, default is `10`
+
+For example, to show results from 20th document to 30th document, set `from: 20` and `size: 10`. For more information about pagination, see [paginate results]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/paginate/#the-from-and-size-parameters).
+
+### The impact of pagination_depth on hybrid search results
+
+The change in `pagination_depth` also affects the search results ordering on which the user is paginating. This is because altering the `pagination_depth` directly impacts the number of results retrieved for each subquery per shard, which may ultimately might change the result ordering after normalization. Therefore, it is recommended to maintain a consistent value of `pagination_depth` while navigating between pages. 
```

Comment:
I don't think we need the end of the first sentence as it feels a little clunky.  How about `The change in pagination_depth also affects the ordering of search results.`

Reply:
Essentially what i want to say here is if you some plays around with pagination_depth then it will change the ground truth on which the user is paginating.

Reply:
Understood, I still think there is probably a better way to phrase it but I can't think of anything.  It's technically sound, so we can leave phrasing to the doc team